plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -62,6 +62,7 @@
     description:
       - Determine if developers can create projects in the group.
     choices: ["developer", "maintainer", "noone"]
+    default: "maintainer"
```

Comment:
Defaults are not acceptable. They cause problems with idempotence (tasks suddenly changing properties of objects that were not specified).

Reply:
not exactly, an idempotency problem exists if you can create it... and for now, it's impossible.

but i understand you point of view : on already created (by hand) this one can change the attribute (because i dont take time to find the default value gitlab choose, feel free to change this default value)

And for the last question : those two attributes are mandatory for creation, but not for update

Reply:
and for the last one : feel free to change the documentation to add those mandatories options for creation and reject this PR it does work too.

Reply:
the last commit resolve the discussion